test/lib/helpers.bats

```diff
@@ -39,6 +39,39 @@ function local_setup {
   assert_failure
 }
 
+@test 'helpers: pathmunge: ensure function is defined' {
+  run type -t pathmunge
+  assert_line 'function'
+}
+
+@test 'helpers: pathmunge: single path' {
+  new_paths='/tmp/fake-pathmunge-path'
+  old_path="${PATH}"
+  pathmunge "${new_paths}"
+  assert_equal "${new_paths}:${old_path}" "${PATH}"
+}
+
+@test 'helpers: pathmunge: single path, with space' {
+  new_paths='/tmp/fake pathmunge path'
+  old_path="${PATH}"
+  pathmunge "${new_paths}"
+  assert_equal "${new_paths}:${old_path}" "${PATH}"
+}
+
+@test 'helpers: pathmunge: multiple paths' {
+  new_paths='/tmp/fake-pathmunge-path1:/tmp/fake-pathmunge-path2'
+  old_path="${PATH}"
+  pathmunge "${new_paths}"
+  assert_equal "${new_paths}:${old_path}" "${PATH}"
+}
+
+@test 'helpers: pathmunge: multiple paths, with space' {
```

Comment:
It would be good to add a negative test, where you're trying to add the same path segment twice, to show that it will only be added once...